AutomatedTesting/Gem/PythonTests/scripting/NodePalette_HappyPath_CanSelectNode.py

```diff
@@ -6,91 +6,92 @@
-    category_index = pyside_utils.find_child_by_hierarchy(tree, CATEGORY)
-    tree.scrollTo(category_index)
-    pyside_utils.item_view_index_mouse_click(tree, category_index)
-    pyside_utils.wait_for_condition(tree.selectedIndexes() and tree.selectedIndexes()[0] == category_index)
-    Report.result(Tests.category_selected, tree.selectedIndexes()[0] == category_index)
-
-    # 5) Click on node and check if it is selected
-    node_index = pyside_utils.find_child_by_pattern(tree, NODE)
-    helper.wait_for_condition(lambda: tree.isExpanded(node_index), GENERAL_WAIT)
-    pyside_utils.item_view_index_mouse_click(tree, node_index)
-    pyside_utils.wait_for_condition(tree.selectedIndexes()[0] == node_index)
-    Report.result(Tests.node_selected, tree.selectedIndexes()[0] == node_index)
+    node_palette = sc.findChild(qtwidgets.QDockWidget, NODE_PALETTE_QT)
+
+    # 3) Get a handle on the node palette's tree of nodes then expand the QTView tree object
+    node_palette_tree_view = node_palette.findChild(qtwidgets.QTreeView, TREE_VIEW_QT)
+    node_palette_tree_view.expandAll()
+
+    # 4) Scroll down to the category we are looking for and verify we can click on it
+    category_index = pyside_utils.find_child_by_hierarchy(node_palette_tree_view, NODE_CATEGORY)
+    node_palette_tree_view.scrollTo(category_index)
+    pyside_utils.item_view_index_mouse_click(node_palette_tree_view, category_index)
+    pyside_utils.wait_for_condition(lambda: node_palette_tree_view.selectedIndexes() and
+                                            (node_palette_tree_view.selectedIndexes()[0] == category_index),
+                                    WAIT_TIME_3)
+    Report.result(Tests.category_selected, node_palette_tree_view.selectedIndexes()[0] == category_index)
+
+    # 5) Scroll down to the node within the category and verify we can click on it
+    pyside_utils.find_child_by_pattern(node_palette_tree_view, NODE_NAME)
+    node_index = pyside_utils.find_child_by_pattern(node_palette_tree_view, NODE_NAME)
+    node_palette_tree_view.scrollTo(node_index)
+    pyside_utils.item_view_index_mouse_click(node_palette_tree_view, node_index)
+    pyside_utils.wait_for_condition(lambda: node_palette_tree_view.selectedIndexes() and
+                                            (node_palette_tree_view.selectedIndexes()[0] == node_index),
+                                    WAIT_TIME_3)
+    Report.result(Tests.node_selected, node_palette_tree_view.selectedIndexes()[0] == node_index)
 
     # 6) Close Script Canvas window
-    general.close_pane("Script Canvas")
-
-
-if __name__ == "__main__":
-    import ImportPathHelper as imports
+    general.close_pane(SCRIPT_CANVAS_UI)
 
-    imports.init()
-    from utils import Report
 
-    Report.start_test(NodePalette_HappyPath_CanSelectNode)
+if __name__ == "__periodic__":
+    from editor_python_test_tools.utils import Report
+    Report.start_test(TestNodePaletteHappyPathCanSelectNode)
```

Comment:
NIT: new line at end of file to follow PEP8 standards

Reply:
added a line to meet the PEPE8 standard